cassandra-analytics-core/src/main/java/org/apache/cassandra/spark/bulkwriter/SqlToCqlTypeConverter.java

```diff
@@ -763,6 +766,12 @@ else if (object instanceof Map)
             throw new RuntimeException("Unsupported conversion for MAP from " + object.getClass().getTypeName());
         }
 
+        @Override
+        public String toString()
+        {
+            return "Map";
```

Comment:
How about the richer string? 

```suggestion
            return "Map<" + keyConverter.toString() + ", " + valConverter.toString() + '>';
```

Reply:
Ha - sure... although technically this PR didn't actually change that, it's just a formatting thing